marimbabot_planning/scripts/dummy_client.py

```diff
@@ -21,19 +26,43 @@ def main_loop(self):
         """
         octave = int(input("Enter an octave (e.g. 4): "))
         while not rospy.is_shutdown():
-            input_str = input("Enter a sequence of hits (e.g. 'C, C#, A'): ")
+            # get input from user and create goal
+            input_str = input("Enter a sequence of hits (e.g. 'C, <C#, D>, C#, A'): ")
             goal = HitSequenceGoal()
             start_time = rospy.Time(0)
             duration = rospy.Duration(0.5)
-            for note in input_str.split(','):
+
+            # create regex for splitting the notes including only the notes from the scale (C, D, E, F, G, A, B)
+            # e.g. 'C, <C#, D>, C#, A' => ['C', '<C#, D>', 'C#', 'A']
+            # LIMITED TO A CHORD OF 2 NOTES !!!
+            regex_pattern = r'([CDEFGAB][#b]?|[<].*?[>])'
```

Comment:
The parser also doesn't check the ,
```
>>> re.findall(regex_pattern, "C#AbC")
['C#', 'Ab', 'C']
```

is this desired?

Reply:
No, I don't think that harms a lot anyways